src/SeaDrop.sol

```diff
@@ -603,7 +603,11 @@ contract SeaDrop is ISeaDrop, ReentrancyGuard {
      * @param endTime   The drop stage end time.
      */
     function _checkActive(uint256 startTime, uint256 endTime) internal view {
-        if (block.timestamp < startTime || block.timestamp > endTime) {
+        if (
```

Comment:
What's reason for adding this?

Reply:
just an optimization to reduce a jumpi, figured it doesn't hurt to have for the future